huggingface_datasets_converter/convert.py

```diff
@@ -92,6 +97,14 @@ def get_kaggle_metadata(kaggle_id):
     except:
         license = 'unknown'
 
+    if license == 'unknown' or license == 'other':
+        raise NameError(
```

Comment:
Should we perhaps add guidance on how you might be able to reach out to the kaggle author to see if they could add a license?

Reply:
Agree. Added this text to the error `"You can ask the dataset author to specify a license in the 'Discussion' section of the dataset's Kaggle page."`

I was not aware there was a Discussion section in Kaggle!